src/droplet.F

```diff
@@ -2201,7 +2201,11 @@ subroutine gauss_newton_2d(diffnorm,dt_nondim,taup_scale,rt_start,rt_zeros,flag,
 
         v1(1) = rt_start(1)
         v1(2) = rt_start(2)
-        fv2 = (/1., 1./)
+         
+        fv2(:) = 1.
```

Comment:
Will it be better to write `fv2(1) = 1.` and  `fv2(2) = 1.`?

Reply:
Just updated the code to eliminate the array syntax